ui/pom.xml

```diff
@@ -30,7 +30,7 @@
     <version>1.21-SNAPSHOT</version>
   </parent>
   <artifactId>application-ckeditor-ui</artifactId>
-  <name>CKEditor Integration UI</name>
+  <name>CKEditor Integration</name>
```

Comment:
Ok I give up I don't know where/how to fix it (I don't understand what I'm doing with jenkins). Now I wonder if href must fixed here on pom because <url>https://github.com/xwiki-contrib/application-ckeditor/tree/master</url> is correct but wrong on exo page

Reply:
The XWiki Repository Application gets the URL from the Maven `<scm><url>` value (see https://github.com/Pbast/application-ckeditor/blob/79369fe19fc91665794c7255f42e516fb9db3d50/pom.xml#L46).